README.md

```diff
@@ -450,17 +451,10 @@ https://github.com/spheromak/dhcp-cook
 
 |                      |                                          |
 |:---------------------|:-----------------------------------------|
+| **Maintainer Community** | [Sous Chefs](https://github.com/sous-chefs)
 | **Maintainer**       | [Jacob McCann](https://github.com/jmccann)
 | **Original Author**  | [Jesse Nelson](https://github.com/spheromak)
 | **Author**           | [Matt Ray](https://github.com/mattray)
-| **Contributor**      | [Myles Steinhauser](https://github.com/masteinhauser)
```

Comment:
Generally speaking, we don't list contributors in any of the cookbooks since the commit history has it